src/Tools/Tests.Common/Tests.Common.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <DefaultItemExcludes>$(DefaultItemExcludes);TestProjects\**\*</DefaultItemExcludes>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="TestContent\*.txt" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="TestProjects\**\*" CopyToOutputDirectory="PreserveNewest" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Properties\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="TestContent\*.txt">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
This is an unusual choice for embedded content. Any reason why it’s both embedded and copied to build output?